edx_rest_framework_extensions/jwt_decoder.py

```diff
@@ -114,9 +114,16 @@ def decode_jwt_scopes(token):
     return jwt_decode_handler(token).get('scopes', [])
 
 
+def decode_jwt_is_restricted(token):
+    """
+    Decode the JWT and return the is_restricted claim.
+    """
+    return jwt_decode_handler(token).get('is_restricted', True)
```

Comment:
Will defaulting to `True` be a problem for existing tokens out in the wild that do not have the `is_restricted` claim?

Reply:
You're right.  In fact, this is a great time where I can use the `version` number on the token to make a safe assumption of whether or not to expect `is_restricted`.  😄